_layouts/events.html

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
do we wanna continue doing this event approach or instead just use the Event category on github  discussions? or some mix?

Reply:
I was thinking a mix since we already have the world tour on the site. We could use the site to list major events we are attending/sponsoring but use the GH Discussion section for additional communication.